src/Util/ColumnParser.php

```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```

Comment:
@josegonzalez I understand then, I found another one in another method that I removed in a previous PR.